packages/pyright-internal/src/analyzer/program.ts

```diff
@@ -187,6 +187,7 @@ export class Program {
     private _cacheManager: CacheManager;
     private _id: number;
     private static _nextId = 0;
+    private _editMode = false;
```

Comment:
Nit: rename to `_isEditMode` to make it obvious this is a boolean and not an enumeration or something like that.